Server/Aujard/AujardDlg.cpp

```diff
@@ -929,39 +923,39 @@ void CAujardDlg::AllSaveRoutine()
 	// log the disconnect
 	msgStr.Format(_T("Ebenezer disconnected: %04d/%02d/%02d %02d:%02d"), cur.GetYear(), cur.GetMonth(), cur.GetDay(), cur.GetHour(), cur.GetMinute());
 	OutputList.AddString(msgStr);
-	TRACE(_T("Dead Time : %04d/%02d/%02d %02d:%02d\n"), cur.GetYear(), cur.GetMonth(), cur.GetDay(), cur.GetHour(), cur.GetMinute());
+	spdlog::error("Ebenezer disconnected. Saving users...");
 	
 	for (int userId = 0; userId < static_cast<int>(_dbAgent.UserData.size()); userId++)
 	{
 		_USER_DATA* pUser = _dbAgent.UserData[userId];
 		if (pUser == nullptr || strlen(pUser->m_id) == 0)
 		{
-#if defined(_DEBUG)
-			TRACE(_T("GameServer Dead!! - %hs Skipped\n"), pUser->m_id);
-#endif
+			spdlog::debug("userId skipped for invalid data: {}", userId);
 			continue;
 		}
 
 		if (HandleUserLogout(userId, UPDATE_ALL_SAVE, true))
 		{
-			TRACE(_T("GameServer Dead!! - %hs Saved\n"), pUser->m_id);
+			spdlog::debug("Character saved: {}", pUser->m_id);
 		}
 		else
 		{
 			allUsersSaved = false;
-			TRACE(_T("GameServer Dead!! - %hs Not Saved\n"), pUser->m_id);
+			spdlog::error("Failed to save character: {}", pUser->m_id);
```

Comment:
Probably still needs context, just so it's clear why they were attempted to be saved. There is the initial line above, but it could be saved concurrently with a regular save so there's no way of telling which triggered it.